modules/statics/src/coins.ts

```diff
@@ -115,6 +115,32 @@ export const coins = CoinMap.fromCoins([
   account('txtz', 'Testnet Tezos', Networks.test.xtz, 6, UnderlyingAsset.XTZ, XTZ_FEATURES),
   account('susd', 'Silvergate USD', Networks.main.susd, 2, UnderlyingAsset.USD),
   account('tsusd', 'Testnet Silvergate USD', Networks.test.susd, 2, UnderlyingAsset.USD),
+  account(
```

Comment:
There are existing examples of both formats (all in one line vs all spread out), is there one in particular the SDK team prefers?

Reply:
The format is forced by prettier (all spread out when the line contains certain amount of characters)

Reply:
ahh, good to know